Source/AssetRipper.SourceGenerated.Extensions/TransitionConstantExtensions.cs

```diff
@@ -60,7 +60,5 @@ public static bool GetHasExitTime(this ITransitionConstant transitionConstant)
 				return false;
 			}
 		}
-
-		public static bool IsExit(this ITransitionConstant transitionConstant) => transitionConstant.DestinationState >= 30000;
```

Comment:
Is this inaccurate?

Reply:
yeah, a `Destination` of >= 30000 means its targeting a (Sub)`StateMachine`'s Entry _or_ Exit point.
Being Entry or Exit depends on the targeted `SelectorStateConstant`'s `IsEntry` bool